src/js/claims-status/components/appeals-v2/Docket.jsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import moment from 'moment';
 
 // TODO: Depending on when this gets rendered, we may want to return null if either
 //  `ahead` or `total` don't exist (or if `total` === 0 because of the divide by 0  error)
-function Docket({ ahead, total }) {
+function Docket({ ahead, total, form9Date }) {
   const completedWidth = { width: `${((total - ahead) / total) * 100}%` };
+  const date = moment(form9Date, 'YYYY-MM-DD').format('MMMM YYYY');
```

Comment:
What happens if `form9Date` is falsey?

Reply:
That's right, without a Form 9, the docket progress bar will not be shown. But check out https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8454. I'm going to add the month into the `docket` object.